SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -38,7 +39,11 @@ void FSpatialGDKEditor::GenerateSchema(FSimpleDelegate SuccessCallback, FSimpleD
 
 	TryLoadExistingSchemaDatabase();
 
-	LoadAllStreamingLevels(GWorld);
+	const USpatialGDKEditorSettings* SpatialGDKSettings = GetDefault<USpatialGDKEditorSettings>();
+	if (SpatialGDKSettings->bLoadStreamingLevelsWhenGeneratingSchema)
```

Comment:
The commandlet will need a way to force/override this setting from the commandline.

TBH, is there even a use case for generating schema WITHOUT sublevels being loaded?

Reply:
Good point, will add something.

Added the flag in case loading all sublevels would take a very long time (if the world is huge), and you're iterating on a small part of the level. Might turn out to be not that useful, but didn't want to break any existing workflows.